packages/focusvisible/src/FocusVisibleContainer.spec.tsx

```diff
@@ -18,7 +18,7 @@ describe('FocusVisibleContainer', () => {
       <FocusVisibleContainer>
         {({ ref }) => (
           <div ref={ref} data-test-id="wrapper">
-            <button data-test-id="button" tabIndex="0"></button>
+            <button data-test-id="button" tabIndex={0}></button>
```

Comment:
The type definition for [`tabIndex`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1670) is of type number.